scripts/build_boost_source.bat

```diff
@@ -26,11 +26,11 @@ if %ERRORLEVEL% NEQ 0 goto :error
 
 set curdir=%cd%
 
-if "%platform%"=="x64" (
+if "%platform%"=="X64" (
```

Comment:
We can't just change that as same checking in other scripts as well like this one for azure sdk: https://github.com/snowflakedb/libsnowflakeclient/blob/v1.0.0/scripts/build_azuresdk.bat#L32
Do we know why it become upper case and fix that directly? Otherwise we need to fix this in all other scripts as well, maybe make the checking case insensitive.

Reply:
I couldn't find where this value was updated to upper case. I am going to review one more time. Otherwise I will fix in the other scripts.

Reply:
Updated the other scripts.